internal/controller/components/datasciencepipelines/datasciencepipelines.go

```diff
@@ -40,13 +41,34 @@ func (s *componentHandler) GetManagementState(dsc *dscv1.DataScienceCluster) ope
 	return operatorv1.Removed
 }
 
-func (s *componentHandler) Init(_ common.Platform) error {
+func getExtraParams(dsp *componentApi.DataSciencePipelines) (map[string]string, error) {
 	release := cluster.GetRelease()
 	clusterInfo := cluster.GetClusterInfo()
+
 	extraParams := map[string]string{
-		platformVersionParamsKey: release.Version.String(),
-		fipsEnabledParamsKey:     strconv.FormatBool(clusterInfo.FipsEnabled),
+		platformVersionParamsKey:          release.Version.String(),
+		fipsEnabledParamsKey:              strconv.FormatBool(clusterInfo.FipsEnabled),
+		argoWorkflowsControllersParamsKey: "{}",
```

Comment:
probably should set this to `"true"`, right?

Reply:
The value of the param is a marshalled JSON of `dsp.Spec.ArgoWorkflowsControllers`. That's why it's an empty dictionary by default. This is needed because when the controller is initialized, `getExtraParams` is called before we have the DSP configuration.